bundle-workflow/src/ci_workflow/check_gradle_publish_to_maven_local.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+from ci_workflow.check import Check
+
+
+class CheckGradlePublishToMavenLocal(Check):
+    def check(self):
+        cmd = " ".join(
+            [
+                "./gradlew publishToMavenLocal",
```

Comment:
You have a point, however this is actually quite far from a full build, only some gradle runs to extract properties. Once we have maven we don't need to publish OpenSearch and common-utils to maven local, so this CI will be much faster. 

If we start building everything we're going to start running out of space on workers, it will take 2 hours on GHA, etc.

Reply:
I added running CI on manifests only in PRs, that's a good idea.